### PR TITLE
Ccs edits 260815

### PR DESCRIPTION
**Pull request by superroz**

### CODE REVIEW

File: g7_declaration/SQ1-1b.yml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 question: What is your organisation's registered office address?
-hint: "If you don't have a registered office address, you don't need to answer this question."
```
Comment (TheDoubleK) [inline g7_declaration/SQ1-1b.yml]: Should there at least be something here to show that the question is optional?

File: g7_declaration/SQ1-1d-i.yml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 question: When was your organisation first registered?
-hint: If you don't have a registered organisation, you don't need to answer this question.
```
Comment (TheDoubleK) [inline g7_declaration/SQ1-1d-i.yml]: And here...

File: g7_declaration/SQ1-1d-ii.yml [hunk 3]
```diff
@@ -1,3 +1,2 @@
 question: In which country was your organisation first registered?
-hint: If you don't have a registered organisation, you don't need to answer this question.
```
Comment (TheDoubleK) [inline g7_declaration/SQ1-1d-ii.yml]: And here?

File: g7_declaration/SQ1-1e.yml [hunk 4]
```diff
@@ -1,6 +1,5 @@
 question: What is your organisation's company registration number?
 hint: >
   If you don't have a Companies House number, provide the company registration
-  number from the country where you first registered. If you don't have an
-  organisation, you don't have to answer this question.
```
Comment (TheDoubleK) [inline g7_declaration/SQ1-1e.yml]: etc.

Comment (superroz) [inline g7_declaration/SQ1-1e.yml]: It’s not optional anymore. Cath has written a story to fix the validation: https://www.pivotaltracker.com/story/show/102066014

Comment (TheDoubleK) [inline g7_declaration/SQ1-1e.yml]: Oh... OK then, I think.

**Comment by TheDoubleK** [comment]
:+1:

**Pull request merged**